### ISSUE TITLE
ColumnToggler: Keyboard trap on Nova and Luna

### ISSUE THREAD

**Issue by MarkusChur** [primefaces/primefaces#4977]
## Summary
ColumnToggler suffers a keyboard trap. When focusing checkboxes of this component with the keyboard you are not able to access other elements on the page.
- Navigation with TAB is restricted to the checkboxes
- It is not possible to leave the overlay with the keyboard
- You can not close the overlay with the keyboard
This can be reproduced in the [showcase](https://www.primefaces.org/showcase/ui/data/datatable/columnToggler.xhtml).

More details on the WCAG success criteria can be found here:
https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-trapping.html

## 1) Environment
- It does not work with 7.0.4
- It does not work in the showcase: https://www.primefaces.org/showcase/ui/data/datatable/columnToggler.xhtml

## 2) Expected behavior
- As a keyboard user I would like to leave the ColumnToggler with the keyboard
- As a keyboard user I would like to close the ColumnToggler overlay
## 3) Actual behavior
- Keyboard trap

## 4) Steps to reproduce
- Navigate to the ColumnToggler checkboxes with the keyboard
- Try to leave the component with the keyboard only
## 5) Sample XHTML
- Please refer to https://www.primefaces.org/showcase/ui/data/datatable/columnToggler.xhtml
## 6) Sample bean
- Please refer to https://www.primefaces.org/showcase/ui/data/datatable/columnToggler.xhtml

**Comment by melloware** [comment]
@mertsincan I think the original design of the column toggler something is broken.  I see it creates a hidden "closer" and assigns the ability to close the toggler but it is not visible.  And it looks like when pressing TAB the intent was after tabbing on the last checkbox it should then focus the "closer" so then you can press ENTER to close the toggler.    However I am not seeing that happen.

I can see the "closer" in the DOM but the code is definitely not doing what I think its original intent was.  Can you check?

**Comment by MarkusChur** [comment]
While working on https://github.com/primefaces/primefaces/issues/4979 I've noticed that the "closer" is displayed, working, and in taborder. I think this one can be closed then.

**Comment by melloware** [comment]
@MarkusChur When I run using the Showcase example its not being displayed and thus the keyboard trap is not working.  What Theme are you using?  

https://www.primefaces.org/showcase/ui/data/datatable/columnToggler.xhtml

I have a feeling a Theme is hiding it.

**Comment by MarkusChur** [comment]
You are totally right, I used a different theme. Switch to `primefaces-omega` with the theme switcher and the "closer" is visible.

**Comment by melloware** [comment]
Yep the premium themes look like they have this issue.

```css
body .ui-columntoggler .ui-columntoggler-close {
    display: none;
 }
```
